src/pages/fundamentals/icons.hbs

```diff
@@ -1,16 +1,13 @@
 ---
-title: Icon Assets
```

Comment:
Note for reviewer: Omitting the title effectively hides this in the nav (the link is still there, it's just empty). This is a total hack but I hope it's short-lived.